index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+const busTimeRequest = require('./api-requests');
+
+// Setup static pages for '/' and '/history'
+app.use(express.static('frontend'));
+app.use('/history', express.static('frontend/history.html'));
+
+// Internal request for bus data
+app.get('/departureBoards', (req, res) => {
+    const postcode = req.query.postcode;
+    console.log(`Request made for postcode '${postcode}'`);
+    
+    if (postcode) {
+        busTimeRequest.getBusArrivalJSON(postcode).then(
+            (json) => {
+            res.send(json);
+        }, (err) => {

```

Comment:
`.then(...).catch(...)` is a little nicer (and generally safer) than `.then(..., ...)`